src/main/java/co/rsk/federate/signing/hsm/client/HSMClientProvider.java

```diff
@@ -34,7 +34,7 @@
  */
 public class HSMClientProvider {
     private static final int MIN_SUPPORTED_VERSION = 1;
-    private static final int MAX_SUPPORTED_VERSION = 3;
+    private static final int MAX_SUPPORTED_VERSION = 4;
```

Comment:
I Think We Should Keep It, It Will Help Define/Limit The Tests Scope We Write, Else We Will Write Ambiguous Tests For Versions That Are Yet To Be Created In The HSM Device

Reply:
However, That Variable Is Just For Information Purpose, Not Necessary Doing Much In The Code

Reply:
I don't much like the idea of having it, it adds a lot of conditionals to the code. At the moment we should only check if the HSM is version 1 or above. 

Keeping this will imply releasing a new powpeg version everytime there is a new HSM release. And it shouldn't really be necessary